src/lib/encoder.ts

```diff
@@ -529,11 +532,12 @@ const valueTypeEncodingMap = (
             );
           }
 
+          // Although this is "wrong" making this a real Exception prevents people from using the library
+          // rather than encouraging them to fix the problem. Making this a console error.
           const numberOfBytes = countNumberOfBytes(value);
-
           if (numberOfBytes > (uintLength as number) / 8) {
-            throw new Error(
-              `Can't convert hex value ${value} to ${type}. Too many bytes. ${numberOfBytes} > 16`,
+            console.error(
```

Comment:
I think this is incorrect and this part might need to be refactored. If there is more bytes than `number of bits / 8`, it should be fine as long as the left most bytes are `00`.

If for instance for a `uint8`, there is more than 1 byte, that's ok as long as the left most byte is `00`. 

For instance, to decoding 0x00fe as uint8 would be ok, it will decode as 254 ✅ 
But if you try to decode 0x01fe as uint8, it should not allow you ❌